en/computer_vision/collision_prevention.md

```diff
@@ -197,10 +185,21 @@ For more information on hardware and software setup see: [PX4/PX4-Avoidance > Ru
 The hardware and software should be set up as described in the [PX4/PX4-Avoidance](https://github.com/PX4/PX4-Avoidance) repo.
 In order to emit `OBSTACLE_DISTANCE` messages you must use the _rqt_reconfigure_ tool and set the parameter `send_obstacles_fcu` to true.
 
-## Gazebo Setup
+## Gazebo Simulation
+
+_Collision Prevention_ can be tested using [Gazebo](../sim_gazebo_gz/index.md) with the [x500_lidar](../sim_gazebo_gz/vehicles.md#x500-quadrotor-with-2d-lidar) model.
+To do this, start a simulation with the x500 lidar model by running the following command:
+
+```sh
+make px4_sitl gz_x500_lidar
+```
+
+Next, adjust the relevant parameters to the appropriate values and add arbitrary obstacles to your simulation world to test the collision prevention functionality.
+
+The diagram below shows how the simulation looks when viewed in RViz.
```

Comment:
Actually its Gazebo and not RViz 😅

Reply:
Thanks @Claudio-Chies, I'll fix - didn't know you could get the sensor view like that. BTW, do you know much about prearm conditions on collision prevention  - got a bug https://github.com/PX4/PX4-Autopilot/issues/22994 and from my comment to that you can see a linked (current) discussion board post.

Reply:
puh, for Collision Prevention, i think you just have to have a obstacle_distance message present. but in the issue he's talking about avoidance, and there its handeled differently from what i know. as it has to wait for the onbaord computer to be ready

Reply:
Ah, we should probably document that :-). So to be clear, the implication is that you need a component that is sending a HEARTBEAT and has type https://mavlink.io/en/messages/common.html#MAV_TYPE_ONBOARD_CONTROLLER right?

Reply:
Ignore that, I can see the code 
```
			case MAV_COMP_ID_OBSTACLE_AVOIDANCE:
				_heartbeat_component_obstacle_avoidance = now;
				_mavlink->telemetry_status().avoidance_system_healthy = (hb.system_status == MAV_STATE_ACTIVE);
				break;
```

Incorrect in that it relies on the companion computer component ID rather than mavlink type
-